resources/build/minimum-versions.inc.sh

```diff
@@ -20,8 +20,7 @@ KEYMAN_MIN_TARGET_VERSION_CHROME=95.0         # Final version that runs on Andro
 # Dependency versions
 KEYMAN_MIN_VERSION_NODE_MAJOR=20              # node version source of truth is /package.json:/engines/node
 KEYMAN_MIN_VERSION_NPM=10.5.1                 # 10.5.0 has bug, discussed in #10350
-KEYMAN_MIN_VERSION_EMSCRIPTEN=3.1.44          # Warning: 3.1.45 is bad (#9529); newer versions work
-KEYMAN_MAX_VERSION_EMSCRIPTEN=3.1.58          # See #9529
+KEYMAN_MIN_VERSION_EMSCRIPTEN=3.1.58          # Use KEYMAN_USE_EMSDK to automatically update to this version
```

Comment:
Does running `publish-minimum-versions.sh` still work? (The Note on l. 8)

Reply:
Yes, oops will run in a follow-up PR. Getting this merged so we can unbreak builds.